src/Repositories/SnapshotRepository.php

```diff
@@ -15,13 +15,23 @@ final class SnapshotRepository
     /** @var array<string, int> */
     private static array $expectationsCounter = [];
 
+    /**
+     * @readonly
+     */
+    private string $testsPath;
+
+    /**
+     * @readonly
+     */
+    private string $snapshotsPath;
+
     /**
      * Creates a snapshot repository instance.
      */
-    public function __construct(
-        readonly private string $testsPath,
-        readonly private string $snapshotsPath,
-    ) {
+    public function __construct(string $testsPath, string $snapshotsPath)
```

Comment:
Why would you change from PHP's built-in `readonly` keyword to a PHPDoc tag? 🤔 I don't think this change is necessary, or makes sense, when only PHP 8.1 and later are supported.